src/structure/widget.ts

```diff
@@ -1007,7 +1007,10 @@ export class MoleculeViewer {
      */
     private _resetView(center: boolean): void {
         if (center && this._highlighted !== undefined) {
-            this._viewer.zoomTo({ serial: this._highlighted.center });
+            // use index rather than serial to specify the selection, to avoid picking also the 
+            // periodic replicas. however we then have to specify the model id, otherwise it'd 
+            // pick the index from the highlighted selection, which does not match the serial ID
+            this._viewer.zoomTo({ index: this._highlighted.center, model: 0 });
```

Comment:
The model number is not guaranteed to be 0, you should use `this._current.model` or something like this

Reply:
actually you were right. works